controllers/volumereplicationgroup_controller.go

```diff
@@ -1019,9 +1025,45 @@ func (v *VRGInstance) updateVRGClusterDataProtectedCondition() {
 		setStatusCondition(&v.instance.Status.Conditions, *volSyncAggregatedCond)
 	}
 
-	if len(v.volRepPVCs) != 0 {
-		v.aggregateVolRepClusterDataProtectedCondition()
```

Comment:
I think removing this also broke VolSync.  `ClusterDataProtected` is meaningless to VolSync.  We don't store anything in the s3Store. To comply with the existing conditions, we set the `ClusterDataProtected` condition to whatever `DataProtected` is set at. It would have been nice to set the status to something like _Not Applicable_ but it is what it is.

Reply:
I wonder if it is necessary to comply with existing conditions.  Is this to satisfy DRPC?  Perhaps DRPC can be modified...  Each condition can be either `false`, `true`, or `unknown`.  I wonder if unknown can be used to represent not applicable?

In PR #531 I propose using a subcondition of nil to indicate not applicable or abstaining from voting.  If all `ClusterDataProtected` subconditions were nil, then it would result in no changes, and that state would remain `unknown`.

Reply:
PR #531 has been merged in to address this.  When there are no volRep PVCs selected, a `nil` volRep `ClusterDataProtected` subcondition is submitted, which is like an abstaining vote.